### PR TITLE
There are timezones that are not round to an hour

### PR DESCRIPTION
**Pull request by Crypto-Python**

### CODE REVIEW

File: src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+return function()
+    local DateTimeOffset = require(script.parent.DateTimeOffset)
+    
+    describe("GetOffset", function()
+        it("should properly return positive full hour", function()
+            local result = DateTimeOffset.GetOffset(SECONDS_IN_HOUR)
+            expect(result).to.equal("UTC+01:00")
+        end)
+
+		it("should properly return negative full hour", function()
```
Comment (vladimirkocovic-htecgroup) [inline src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua]: indentation?

File: src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            local result = DateTimeOffset.GetOffset(-SECONDS_IN_HOUR)
+            expect(result).to.equal("UTC-01:00")
+        end)
+
+        it("should properly return positive partial hour", function()
+            local result = DateTimeOffset.GetOffset(SECONDS_IN_HOUR + SECONDS_IN_MINUTE)
+            expect(result).to.equal("UTC+01:01")
+        end)
+
+		it("should properly return negative partial hour", function()
```
Comment (vladimirkocovic-htecgroup) [inline src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua]: same here

File: src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            local result = DateTimeOffset.GetOffset(-(SECONDS_IN_HOUR + SECONDS_IN_MINUTE))
+            expect(result).to.equal("UTC-01:01")
+        end)
+
+		it("should properly return zero offset", function()
```
Comment (vladimirkocovic-htecgroup) [inline src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua]: and here

**Review by vladimirkocovic-htecgroup — APPROVED** [review]
looks good

File: src/ReplicatedStorage/Utils/DateTimeOffset.lua [hunk 4]
```diff
@@ -1,29 +1,38 @@
 local DateTimeOffset = {}
+local SECONDS_IN_MINUTE = 60
+local SECONDS_IN_HOUR = 60 * SECONDS_IN_MINUTE
+
+function DateTimeOffset.GetOffset(seconds)
+	if seconds == 0 then
+		return "UTC00:00"
```
Comment (MrBotBuzzAlot) [inline src/ReplicatedStorage/Utils/DateTimeOffset.lua]: I think the `Z` timezone should also have a sign. Usually, it is displayed as `UTC+00:00`.

Comment (Crypto-Python) [reply src/ReplicatedStorage/Utils/DateTimeOffset.lua]: Done.

File: src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua (hunk 3 above)
Comment (Crypto-Python) [reply src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua]: You're right. I looked at it in Visual Studio Code, and it looked ok. Used spaces now.

File: src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua (hunk 2 above)
Comment (Crypto-Python) [reply src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua]: done

File: src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua (hunk 1 above)
Comment (Crypto-Python) [reply src/ReplicatedStorage/Utils/DateTimeOffset.spec.lua]: done

**Pull request merged**